components/Partners/Partners.css

```diff
@@ -0,0 +1,7 @@
+.partnerLogos {
+  display: flex;
+  align-self: center;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
```

Comment:
space evenly doesn't have great browser support, let's do `space-between` instead.

Reply:
added space-between as fallback, thanks for the tip